includes/admin/class-wc-rest-payments-payment-intents-controller.php

```diff
@@ -46,4 +74,47 @@ public function get_payment_intent( $request ) {
 
 		return $this->forward_request( 'get_intent', [ $payment_intent_id ] );
 	}
+
+	/**
+	 * Create a payment intent.
+	 *
+	 * @param WP_REST_Request $request data about the request.
+	 */
+	public function post_payment_intent( $request ) {
```

Comment:
Nitpick. I would rename this function to create_payment_intent, but feel free my comment.

Reply:
Done